modules/op-advantages-and-disadvantages-of-non-versioned-and-versioned-cluster-tasks.adoc

```diff
@@ -26,7 +26,7 @@ a|
 a| 
 * If you continue using an earlier version of a cluster task, you might miss the latest features and critical security updates.
 * The earlier versions of cluster tasks that are not operational consume cluster resources.  
-* When upgraded , the Operator cannot manage the earlier VCT. You can delete the earlier VCT manually using the `oc delete clustertask` command, but you cannot restore it.  
+* When upgraded, the Operator cannot manage the earlier VCT. You can delete the earlier VCT manually using the `oc delete clustertask` command, but you cannot restore it.  
```

Comment:
To reviewers: Resolved the comment https://github.com/openshift/openshift-docs/pull/45917#discussion_r889026309. This has no relation with release notes.